src/me/StevenLawson/TotalFreedomMod/Commands/Command_spectator.java

```diff
@@ -0,0 +1,73 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.OP, source = SourceType.BOTH)
+@CommandParameters(description = "Quickly change your own gamemode to spectator, or define someone's username to change theirs.", usage = "/<command> [partialname]", aliases = "gmsp")
+public class Command_spectator extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (senderIsConsole)
+        {
+            if (args.length == 0)
+            {
+                sender.sendMessage("When used from the console, you must define a target user to change gamemode on.");
+                return true;
+            }
+        }
+
+        Player player;
+        if (args.length == 0)
+        {
+            player = sender_p;
+        }
+        else
+        {
+        // Not like we will ever use this but..
+            if (args[0].equalsIgnoreCase("-a"))
```

Comment:
Depending on what happens with my previous comment (Permissions wise) this may be smart to remove (If it is admin only)